php_fpm/tox.ini

```diff
@@ -16,22 +16,11 @@ passenv =
 deps =
     -e../datadog_checks_base[deps]
     -rrequirements-dev.txt
-
-[testenv:unit]
-commands =
-    pip install -r requirements.in
-    pytest -v -m "not integration"
-
-[testenv:py27-integration]
-commands =
-    pip install -r requirements.in
-    pytest -v -m "integration"
-
-[testenv:py36-integration]
 commands =
     pip install -r requirements.in
-    pip install flup==1.0.3
-    pytest -v -m "integration"
+    unit: pytest -v -m "not integration"
```

Comment:
Do we need `-v` here? We should also start testing even `unit` with PY3

Reply:
Yeah we should keep `-v` everywhere for people running tests directly with tox